src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -232,11 +232,13 @@ public class EmptyStringToCollapsedConverter : IValueConverter
         }
     }
 
+    // Rewrite converter when Addons treeview will be visible.
+    // Task: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6226.
```

Comment:
I have added a comment to this task, and make it `Major`. Thanks for noting it down.